<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listeBatiments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonAfficherDetails">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>80</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les détails</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonDemarrerCycle">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Démarrer un cycle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonSupprimerBatiment">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>120</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Supprimer un bâtiment</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressEau">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Eau</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressElectricite">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>5000</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressSatisfaction">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>190</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Electricité</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>160</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Satisfaction</string>
    </property>
   </widget>
   <widget class="QLabel" name="nomville">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="villedetails">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>40</y>
      <width>421</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>250</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Maison</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>381</width>
        <height>251</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="Ajoutermaison">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter maison</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxCapaciteHabitants">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>160</y>
          <width>44</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>CapaciteHabitants :</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxHabitant">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>120</y>
          <width>44</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>71</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Habitant : </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Electricité :</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxElectricite">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>44</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBoxEau">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>40</y>
          <width>44</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximum">
         <number>10000</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Eau :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditNom">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>10</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Nom : </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Usine</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>531</width>
        <height>241</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="Ajouterusine">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter usine</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Parc</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>541</width>
        <height>241</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QPushButton" name="Ajouterparc">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>121</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter Parc</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="Buttonbatiment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>271</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher les données de ce batiment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boutonAjouterBatiment">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter un bâtiment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditDetails">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>419</width>
      <height>177</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
